main/src/main/java/org/mobilitydata/gtfsvalidator/validator/FareAttributeAgencyIdValidator.java

```diff
@@ -43,10 +43,11 @@ public class FareAttributeAgencyIdValidator extends FileValidator {
 
   @Override
   public void validate(NoticeContainer noticeContainer) {
-
+    int totalAgencies = 0;
     // routes.agency_id is required when there are multiple agencies
-    int totalAgencies = agencyTable.entityCount();
-
+    if (agencyTable != null) {
+      totalAgencies = agencyTable.entityCount();
+    }
```

Comment:
[suggestion]: Have this code in its own PR with similar protecting conditionals added to this PR.